### PR TITLE
✨ keybinds

### PR DESCRIPTION
**Pull request by danton267**
Makes use of `dash_extensions` with a simple `EventListener` component defied do listen to keys (can listed to multiple events), and then used as a trigger for a callback where with a bit of extra logic you can write that if key X was pressed do something

### CODE REVIEW

File: components/control_bar.py [hunk 1]
```diff
@@ -250,5 +251,15 @@ def layout():
...
             dcc.Store(id="project-data"),
             html.Div(id="dummy-output"),
+            EventListener(
+                events=[
+                    {
+                        "event": "keydown",
+                        "props": ["key"],
+                    }
+                ],
+                logging=True,
```
Comment (hannahker) [inline components/control_bar.py:261]: Probably want to turn this off since it's just useful for dev

**Comment by hannahker** [comment]
Feel free to merge once merge conflicts have been resolved.

**Comment by danton267** [comment]
- [ ] allow key binds when the sidebar is closed
- [ ] add notification on color change
  - [ ] add class name into notification

**Comment by tamidodo** [comment]
There's an extra line for the line annotation mode at the end of the annotation keybind instructions and a typo: "Keybding Shortcuts" instead of "Keybinding Shortcuts" 
Screenshot 2023-08-10 at 12 38 28 PM

**Review by MinaMirz — CHANGES_REQUESTED** [review]
shift+[1-9] doesn't seem to work on Safari. Everything else was functional.

**Review by MinaMirz — APPROVED** [review]

**Comment by danton267** [comment]
Had to remove alt keys/shift/ctrl because of issues with browser support and cross-compatibility between OS

- ctrl is shared between win and mac but on windows browsers ctrl+1 changes to the first tab of the browser
- alt/option is not shared
- holding shift changes 123456789 to !"£$%^&*( 

if this is requested in the future, we can ask for the browser header to determine what OS/browser user is using and tell app to listen/react to different keys

**Pull request merged**